Configuration.props

```diff
@@ -80,7 +80,7 @@
     <AntDirectory Condition=" '$(AntDirectory)' == '' ">$(AndroidToolchainDirectory)\ant</AntDirectory>
     <AntToolPath Condition=" '$(AntToolPath)' == '' ">$(AntDirectory)\bin</AntToolPath>
     <AndroidSupportedHostJitAbis Condition=" '$(AndroidSupportedHostJitAbis)' == '' ">$(HostOS)</AndroidSupportedHostJitAbis>
-    <AndroidSupportedTargetJitAbis Condition=" '$(AndroidSupportedTargetJitAbis)' == '' ">armeabi-v7a:arm64-v8a:x86</AndroidSupportedTargetJitAbis>
+    <AndroidSupportedTargetJitAbis Condition=" '$(AndroidSupportedTargetJitAbis)' == '' ">armeabi-v7a:arm64-v8a:x86:x86_64</AndroidSupportedTargetJitAbis>
```

Comment:
I was getting build errors in the EmbeddedDSOTests about x86_64 stuff missing.

I am a little confused, but it seems `bin\BuildDebug\XABuildConfig.cs` seems to be the cause:
```csharp
using System;
using System.Collections.Generic;

namespace Xamarin.Android.Tools
{
        static class XABuildConfig
        {
                public const string SupportedABIs = "armeabi-v7a;arm64-v8a;x86;x86_64";
...
```

There might be something else we should do instead here. Let's see if it fixes the tests on CI?